CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ if(WIN32)
 endif()
 
 if(APPLE)
-    set(FreeOrionSDK_VERSION 10)
+    set(FreeOrionSDK_VERSION 11)
```

Comment:
might want to tweak the title if this is also updating to the new SDK for the XCode build...